mitiq/zne/scaling/folding.py

```diff
@@ -67,10 +67,11 @@ def _check_foldable(circuit: Circuit) -> None:
         )
 
     if not has_unitary(circuit):
-        raise UnfoldableCircuitError(
-            "Circuit contains non-unitary channels which are not terminal "
-            "measurements and cannot be folded."
-        )
+        if inverse(circuit, None) is None:
```

Comment:
```suggestion
        if inverse(circuit, default=None) is None:
```
Just to be more explicit and readable.
I didn't know `inverse` had a default option, nice!